docs(windows): add knowledge base article for distributed keyboard installation (#1049)
keymanapp/help.keyman.com: knowledge-base/kb0112.md

**Procedure:**

1. **Keyboard Package Installation:**
   - Execute the following command in an elevated command prompt to install the package and make it available for all users:
     ```
     kmshell -s -i filename.kmp
     ```

2. **Language registration for each user:**
   - With Keyman 14 and above, an extra step is necessary for each user to set up language registration for the keyboard within the package.
   - Ensure the previous kmshell process has completed before running the next command.
   - Run the following command in a non-elevated command prompt for each user:
     ```
     kmshell -s -ikl KeyboardID BCP47Tag
     ```
     - Replace 'KeyboardID' with the appropriate identifier for the keyboard (usually the same as the package filename without `.kmp`).
     - Replace 'BCP47Tag' with the language tag, which can be found in Keyman Configuration.

3. **Example:**
   - As an example, let's consider the installation of the BJ-OjiCree keyboard (Kingfisher):
     - KeyboardID: `bj_cree_east`
     - BCP47Tag: `crl-Cans-CA`
   - Installation commands:
     - Elevated session:
       ```

>        kmshell -s -i bj_cree_east.kmp.kmp

Reviewer: which `.kmp`s should be here?

Author: thanks good pick up I was mergeing to examples.

Author: I also added a description on how to identify the keyboard id and language name. I aslo fixed the last sentence of this  md

Reviewer: gotcha. so should this be bj_cree_east.kmp?

> # Keyman for Windows: Distributed Installation of Keyboards

Reviewer 2: ```suggestion
# Keyman for Windows: Scripted Installation of Keyboards
```

## Installing Keyman Package Files using kmshell.exe

**Introduction:**

> This knowledge base provide the steps to to install keyboards remotely and enable them for a non-elevated user. The key is that the registration for language and the keyboard needs to be run as the user who needs the keyboard enabled.

Reviewer 2: ```suggestion
This article documents the process for installation of a Keyman keyboard package via command line or script. This is particularly helpful when deploying keyboards to multiple computers. The process is split into an elevated step to install the keyboard and register it in the system, and a current-user step, to link the keyboard to a language for the user.
```

**Locating `kmshell.exe`:**
   - Typically, `kmshell.exe` is located in `%ProgramFiles(x86)%\Keyman\Keyman Desktop\kmshell.exe`.
   - Alternatively, the path can be retrieved from the registry setting `HKLM\Software\WOW6432Node\Keyman\Keyman Desktop\root`.

**Procedure:**

1. **Keyboard Package Installation:**
   - Execute the following command in an elevated command prompt to install the package and make it available for all users:
     ```
     kmshell -s -i filename.kmp
     ```

2. **Language registration for each user:**
   - With Keyman 14 and above, an extra step is necessary for each user to set up language registration for the keyboard within the package.
   - Ensure the previous kmshell process has completed before running the next command.
   - Run the following command in a non-elevated command prompt for each user:
     ```
     kmshell -s -ikl KeyboardID BCP47Tag
     ```
     - Replace 'KeyboardID' with the appropriate identifier for the keyboard (usually the same as the package filename without `.kmp`).
     - Replace 'BCP47Tag' with the language tag, which can be found in Keyman Configuration.

3. **Example:**

>    - As an example, let's consider the installation of the **BJCreeUNI** (east) keyboard.

Reviewer 2: ```suggestion
   - As an example, let's consider the installation of the **BJCreeUNI (east)** keyboard.
```

>      Normally, the KeyboardID will be the same as the package filename (sans .kmp). You can check it by installing the package locally and viewing the details in Keyman Configuration. Similarly, the BCP 47 tag (e.g.en-US) can be found in Keyman Configuration installed keyboard’s details looking at the languages label.

Reviewer 2: ```suggestion
     Normally, the KeyboardID will be the same as the package filename (sans .kmp). You can check it by installing the package locally and viewing the details in Keyman Configuration. Similarly, the BCP 47 tag (e.g.en-US) can be found in Keyman Configuration installed keyboard’s details, looking at the languages label.
```
